README.md

```diff
@@ -111,13 +128,15 @@ The `Fixture` class provides a predefined set of generators that supports each t
 For most cases this is fine, and offers a fast and easy way to create fixtures.
 But, for those times where you need a custom implementation, you can write your own [Generator](#generators) to change it's behavior using the `extend` method.
 
-In the example below we create a custom implemantion `AddressGenerator` to return a custom address object.
+In the example below we create a custom implemantion `addressGenerator` to return a custom address object, and a `totalVisitsGenerator` to return a more realistic number of visits.
 
-<sub>[view implementation](./examples/create-fixture-using-generators-person.test.ts)</sub>
+<sub>[view implementation](https://github.com/timdeschryver/zod-fixture/tree/beta/examples/create-fixture-using-generators-person.test.ts)</sub>
 
-```ts
-// examples/create-fixture-extension.test.ts#L2-L36
+::: code-group
 
+<!-- embedme examples/create-fixture-extension.test.ts#L2-L18 -->
```

Comment:
I agree with the comments, and it made also me realize that the readme wouldn't be readable enough via markdown/GitHub.
I got carried away by vitepress 😅

Using a different `embedme` syntax it's possible to use a combination of both solutions, which I like. This makes the readme still readable, while providing a better experience via the docs site.

WDYT?

I think this is the last "blocker" until we can release, and add more docs sections.
Also, could you take a look at your twitter messages when you have the time please?

Reply:
I find the ::: not ideal, but good enough. At least for now